index.html

```diff
@@ -28,44 +28,40 @@
     </header>
     
     <main>   
-    		<!-- Logo top left corner -->
-	    <div id="logo"> 
-		<img src="http://via.placeholder.com/150x150"> 
-	    </div> 
+    	    <!-- Logo top left corner -->
+	    <img id="logo"> 
+		    <img src="http://via.placeholder.com/150x150"> 
+        </img>
 
-		<!-- header/instructions Placeholder -->
-	    <div id="instructions" class="jumbotron">
-		<div id="jumboContainer" class="container">
-		  <h1 class="display-4"></h1>
-		  <p class="Plantinstructions"></p>
-		</div>
-	    </div>
+		    <!-- header/instructions Placeholder -->
+	    <article id="instructions" class="jumbotron">
+            <article id="jumboContainer" class="container">
+                <h1 class="display-4"></h1>
+                <p class="Plantinstructions"></p>
+            </article>
+	    </article>
 
+		    <!-- modal -->
+	    <section class="modal" title="Welcome to Green Thumb">
+            <p>We are so excited to have you here to learn about gardening!</p>
+            <p>Are you ready to become a Green Thumb?</p>
+	    </section>
 
-		<!-- modal -->
-	     <section class="modal" title="Welcome to Green Thumb">
-		<p>We are so excited to have you here to learn about gardening!</p>
-		<p>Are you ready to become a Green Thumb?</p>
-	     </section>
-
-
-
-
-		<!--The div element for the map -->
+		    <!--The div element for the map -->
 	    <form>
-		<label for="zip">Zipcode:</label><br>
-		<input id="zip" name="zip" type="text" pattern="[0-9]*">
-		<input type="submit" value="Submit">
-	    </form>
-	    <div id="localgardenmap"></div>
+            <label for="zip">Zipcode:</label><br>
+            <input id="zip" name="zip" type="text" pattern="[0-9]*">
+            <input type="submit" value="Submit">
+        </form>
+	    <article id="localgardenmap"></article>
```

Comment:
I think this is fine as a div until Sam gives us the map. Article is not an appropriate tab for this.